.github/workflows/bundle_spec_test.yml

```diff
@@ -0,0 +1,43 @@
+on:
+  workflow_run:
+    workflows: ["Create and publish a Docker image"]
+    types: [completed]
+
+name: run bundler test spec
+
+jobs:
+  rub:
+    runs-on: ubuntu-latest
+    steps:
+
+      - run: curl -sSL https://raw.githubusercontent.com/pdm-project/pdm/main/install-pdm.py | python3 -
+
+      - run: pip install jq yq
+
+      - uses: actions/checkout@v3
+        with: 
+          repository: 'https://github.com/eth-infinitism/bundler-test-executor.git'
+
+      - run: test -d bundler-spec-tests || git clone --recurse-submodules --depth 1 https://github.com/eth-infinitism/bundler-spec-tests 
+        name: clone bundler-spec-tests
+
+      - run: cd bundler-spec-tests && (git describe --tags; git submodule ) | tee .git.status
+        name: check bundler-spec-tests and submodules status
+
+      # restore cache of bundler-spec-tests, and its submodules
+      - uses: actions/cache@v2
+        with:
+          path: bundler-spec-tests
+          key: ${{ runner.os }}-${{ hashFiles('bundler-spec-tests/pdm.lock') }}
+
+      - run: cd bundler-spec-tests; git checkout
+        name: "re-checkout bundler-spec-tests (on top of cache)"
+
+        #for faster "update-deps" for spec
+      - run: yarn --cwd bundler-spec-tests/spec remove gatsby || echo "already removed"
+
+      - run: "cd bundler-spec-tests && pdm install && pdm update-deps"
+        name: update submodules of bundler-spec-tests
+
+      - run: ./runall.sh launchers/aabundler-launcher.sh
```

Comment:
Shouldn't this be `./runall.sh launchers/aa-bundler-rust-launcher.sh` ?

Reply:
You are right